02week/bio/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE>
+<html lang="en">
+<head>
+  <title> Isela Triana Biography </title>
+  <link href="css/style.css" rel="stylesheet" type="text/css"/>
+
+<style>
+a:hover{
+  background-color:blue;
+}
+</style>
+</head>
+  <h1> Isela's Bio </h1>
+  <img src = "https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAA1KAAAAJDM3Mzc0ZTRhLTQ0YjItNDZiOC04NmRjLTNiYTczMmFlMzMzNQ.jpg"/>
```

Comment:
Avoid spaces in between attribute and value
No - `src = "..."`
Yes - `src="..."`